operations/operations/company-agreements.md

```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
 
 * If you are on-site and need to sign an NDA to enter a facility you may sign such an agreement 
 * The following agreements may be signed by VP Finance Aneal Vallurupalli 
-  * Non-clickthrough customer agreements up to $5,000 USD 
-  * Procurement contracts for software subscriptions, market events, marketing and advertising agency services up to $5,000 USD 
-  * Sponsored open source projects and paid simulations up to $600 USD
+  * Non-clickthrough customer agreements up to $50,000 USD 
+  * Procurement contracts for software subscriptions, market events, marketing and advertising agency services up to $50,000 USD 
   * Unmodified Mattermost-authored Non-Disclosure Agreements 
   * Unmodified Mattermost-authored Partner Agreements 
 
+Use the [Authorization Matrix](https://docs.google.com/spreadsheets/d/1fDIMiO0uydB_1zCUxZ4sGfSnBJ0P_49zbeQGgTqbYPI/edit?usp=sharing) for a full list of document types and threshold levels 
+
 ## Who are Contract Accountability Owners \(CAOs\)? 
 
 A Contact Accountability Owners, or "CAO", is a Mattermost staff member approved to review and initial contracts of specific types ahead of electronic or physical signature on behalf of the company by the CEO. 
 
 CAOs by area include: 
 
-* Melissa Warner \(Sales Ops\) 
-* Natalie Jew \(HR\)
-* Aneal Vallurupalli \(Finance\)
-* Tim Quock \(Accounting and Procurement\) 
+* Natalie Jew \(HR\) - employment contracts, benefit contracts, vendor contracts (HR) 
+* Aneal Vallurupalli \(Finance\) - sales contracts, corporate contracts, vendor contracts
+* Tim Quock \(Accounting and Procurement\) - sales contracts, corporate contracts, vendor contracts
+* VP \([MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence#mattermost-leadership-team-mlt)\) - vendor contracts
```

Comment:
I don't think MLT should be CAO. 

1. I'd prefer someone from G&A to initial the initial vendor agreement so we have clarity terms for things like indemnity, termination for convenience, spending within budget, etc. are properly reviewed. 

2. I'm fine having MLT initial in addition to a CAO, for things like spend over $50K, etc. but we need someone trained in contract review as CAO.

Reply:
```suggestion

```